roachtest: scrub/index-only/tpcc/w=1000 failed

SHA: https://github.com/cockroachdb/cockroach/commits/3a7ea2d8c9d4a3e0d97f8f106fcf95b3f03765ec

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=scrub/index-only/tpcc/w=1000 PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1187480&tab=buildLog

```
The test failed on master:
	scrub.go:83,cluster.go:1605,errgroup.go:57: pq: communication error: rpc error: code = Canceled desc = context canceled
	cluster.go:1267,tpcc.go:130,cluster.go:1605,errgroup.go:57: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1187480-scrub-index-only-tpcc-w-1000:5 -- ./workload run tpcc --warehouses=1000 --histograms=logs/stats.json --wait=false --tolerate-errors --ramp=5m0s --duration=2h0m0s {pgurl:1-4} returned:
		stderr:
		
		stdout:
		l
		 1h6m24s       40            0.0            2.7      0.0      0.0      0.0      0.0 delivery
		 1h6m24s       40            0.0           27.1      0.0      0.0      0.0      0.0 newOrder
		 1h6m24s       40            0.0            2.7      0.0      0.0      0.0      0.0 orderStatus
		 1h6m24s       40            0.0           25.8      0.0      0.0      0.0      0.0 payment
		 1h6m24s       40            0.0            2.7      0.0      0.0      0.0      0.0 stockLevel
		_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)
		 1h6m25s       40            0.0            2.7      0.0      0.0      0.0      0.0 delivery
		 1h6m25s       40            1.0           27.1 103079.2 103079.2 103079.2 103079.2 newOrder
		 1h6m25s       40            0.0            2.7      0.0      0.0      0.0      0.0 orderStatus
		 1h6m25s       40            0.0           25.8      0.0      0.0      0.0      0.0 payment
		 1h6m25s       40            1.0            2.7    604.0    604.0    604.0    604.0 stockLevel
		: signal: killed
	cluster.go:1626,tpcc.go:140,scrub.go:58,test.go:1214: Goexit() was called

```

SHA: https://github.com/cockroachdb/cockroach/commits/dfa23c01e4ea39b19ca8b2e5c8a4e7cf9b9445f4

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=scrub/index-only/tpcc/w=1000 PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1189954&tab=buildLog

```
The test failed on master:
	cluster.go:1267,tpcc.go:130,cluster.go:1605,errgroup.go:57: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1189954-scrub-index-only-tpcc-w-1000:5 -- ./workload run tpcc --warehouses=1000 --histograms=logs/stats.json --wait=false --tolerate-errors --ramp=5m0s --duration=2h0m0s {pgurl:1-4} returned:
		stderr:
		
		stdout:
		      0.0      0.0 orderStatus
		   4m24s    62311            0.0            0.0      0.0      0.0      0.0      0.0 payment
		   4m24s    62311            0.0            0.0      0.0      0.0      0.0      0.0 stockLevel
		E190321 15:13:11.579140 1 workload/cli/run.go:420  error in payment: dial tcp 10.142.0.181:26257: connect: connection refused
		_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)
		   4m25s   120973            0.0            0.0      0.0      0.0      0.0      0.0 delivery
		   4m25s   120973            0.0            0.0      0.0      0.0      0.0      0.0 newOrder
		   4m25s   120973            0.0            0.0      0.0      0.0      0.0      0.0 orderStatus
		   4m25s   120973            0.0            0.0      0.0      0.0      0.0      0.0 payment
		   4m25s   120973            0.0            0.0      0.0      0.0      0.0      0.0 stockLevel
		E190321 15:13:12.579142 1 workload/cli/run.go:420  error in newOrder: dial tcp 10.142.0.181:26257: connect: connection refused
		: signal: killed
	cluster.go:1626,tpcc.go:140,scrub.go:58,test.go:1214: unexpected node event: 3: dead

```

SHA: https://github.com/cockroachdb/cockroach/commits/7bc9ea5fbe0c0082fdcfd408245a79c62b00edd4

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=scrub/index-only/tpcc/w=1000 PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1197065&tab=buildLog

```
The test failed on master:
	cluster.go:1267,tpcc.go:130,cluster.go:1605,errgroup.go:57: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1197065-scrub-index-only-tpcc-w-1000:5 -- ./workload run tpcc --warehouses=1000 --histograms=logs/stats.json --wait=false --tolerate-errors --ramp=5m0s --duration=2h0m0s {pgurl:1-4} returned:
		stderr:
		
		stdout:
		     0.0      0.0 orderStatus
		   6m44s    75402            0.0            0.0      0.0      0.0      0.0      0.0 payment
		   6m44s    75402            0.0            0.0      0.0      0.0      0.0      0.0 stockLevel
		E190325 14:57:40.401105 1 workload/cli/run.go:420  error in newOrder: dial tcp 10.142.0.127:26257: connect: connection refused
		_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)
		   6m45s   132246            0.0            0.0      0.0      0.0      0.0      0.0 delivery
		   6m45s   132246            0.0            0.0      0.0      0.0      0.0      0.0 newOrder
		   6m45s   132246            0.0            0.0      0.0      0.0      0.0      0.0 orderStatus
		   6m45s   132246            0.0            0.0      0.0      0.0      0.0      0.0 payment
		   6m45s   132246            0.0            0.0      0.0      0.0      0.0      0.0 stockLevel
		E190325 14:57:41.401114 1 workload/cli/run.go:420  error in newOrder: dial tcp 10.142.0.127:26257: connect: connection refused
		: signal: killed
	scrub.go:83,cluster.go:1605,errgroup.go:57: dial tcp 35.229.86.89:26257: connect: connection refused
	cluster.go:1626,tpcc.go:140,scrub.go:58,test.go:1214: unexpected node event: 2: dead

```

SHA: https://github.com/cockroachdb/cockroach/commits/23f9707873abbd2de91a42055535529d7ff296ce

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=scrub/index-only/tpcc/w=1000 PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1209900&tab=buildLog

```
The test failed on release-19.1:
	scrub.go:83,cluster.go:1631,errgroup.go:57: dial tcp 35.243.230.156:26257: connect: connection refused
	cluster.go:1293,tpcc.go:130,cluster.go:1631,errgroup.go:57: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1209900-scrub-index-only-tpcc-w-1000:5 -- ./workload run tpcc --warehouses=1000 --histograms=logs/stats.json --wait=false --tolerate-errors --ramp=5m0s --duration=2h0m0s {pgurl:1-4} returned:
		stderr:
		
		stdout:
		.0      0.0 newOrder
		  10m50s   131747            0.0            0.0      0.0      0.0      0.0      0.0 orderStatus
		  10m50s   131747            0.0            0.0      0.0      0.0      0.0      0.0 payment
		  10m50s   131747            0.0            0.0      0.0      0.0      0.0      0.0 stockLevel
		E190330 14:58:25.742141 1 workload/cli/run.go:420  error in newOrder: dial tcp 10.142.0.33:26257: connect: connection refused
		  10m51s   199449            0.0            0.0      0.0      0.0      0.0      0.0 delivery
		  10m51s   199449            0.0            0.0      0.0      0.0      0.0      0.0 newOrder
		  10m51s   199449            0.0            0.0      0.0      0.0      0.0      0.0 orderStatus
		  10m51s   199449            0.0            0.0      0.0      0.0      0.0      0.0 payment
		  10m51s   199449            0.0            0.0      0.0      0.0      0.0      0.0 stockLevel
		E190330 14:58:26.742155 1 workload/cli/run.go:420  error in payment: dial tcp 10.142.0.33:26257: connect: connection refused
		: signal: killed
	cluster.go:1652,tpcc.go:140,scrub.go:58,test.go:1223: Goexit() was called
	cluster.go:953,context.go:90,cluster.go:942,asm_amd64.s:522,panic.go:397,test.go:774,test.go:760,cluster.go:1652,tpcc.go:140,scrub.go:58,test.go:1223: dead node detection: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod monitor teamcity-1209900-scrub-index-only-tpcc-w-1000 --oneshot --ignore-empty-nodes: exit status 1 5: skipped
		1: dead
		2: 3995
		3: 3849
		4: 3023
		Error:  1: dead

```

SHA: https://github.com/cockroachdb/cockroach/commits/b88a6ce86bfe507e14e1e80fdaefd219b5f0b046

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=scrub/index-only/tpcc/w=1000 PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1222890&tab=buildLog

```
The test failed on release-19.1:
	scrub.go:83,cluster.go:1667,errgroup.go:57: read tcp 172.17.0.2:38340->35.227.66.202:26257: read: connection reset by peer
	cluster.go:1329,tpcc.go:130,cluster.go:1667,errgroup.go:57: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1222890-scrub-index-only-tpcc-w-1000:5 -- ./workload run tpcc --warehouses=1000 --histograms=logs/stats.json --wait=false --tolerate-errors --ramp=5m0s --duration=2h0m0s {pgurl:1-4} returned:
		stderr:
		
		stdout:
		     0.0 newOrder
		   55m5s    52821            0.0            2.0      0.0      0.0      0.0      0.0 orderStatus
		   55m5s    52821            0.0           18.3      0.0      0.0      0.0      0.0 payment
		   55m5s    52821            0.0            1.9      0.0      0.0      0.0      0.0 stockLevel
		E190405 15:21:31.719143 1 workload/cli/run.go:420  error in newOrder: dial tcp 10.142.0.106:26257: connect: connection refused
		   55m6s   103640            0.0            2.0      0.0      0.0      0.0      0.0 delivery
		   55m6s   103640            0.0           19.6      0.0      0.0      0.0      0.0 newOrder
		   55m6s   103640            0.0            2.0      0.0      0.0      0.0      0.0 orderStatus
		   55m6s   103640            0.0           18.3      0.0      0.0      0.0      0.0 payment
		   55m6s   103640            0.0            1.9      0.0      0.0      0.0      0.0 stockLevel
		E190405 15:21:32.719177 1 workload/cli/run.go:420  error in newOrder: dial tcp 10.142.0.106:26257: connect: connection refused
		: signal: killed
	cluster.go:1688,tpcc.go:140,scrub.go:58,test.go:1228: Goexit() was called
	cluster.go:953,context.go:90,cluster.go:942,asm_amd64.s:522,panic.go:397,test.go:776,test.go:762,cluster.go:1688,tpcc.go:140,scrub.go:58,test.go:1228: dead node detection: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod monitor teamcity-1222890-scrub-index-only-tpcc-w-1000 --oneshot --ignore-empty-nodes: exit status 1 5: skipped
		1: dead
		4: 3741
		3: 3938
		2: 3628
		Error:  1: dead

```

SHA: https://github.com/cockroachdb/cockroach/commits/682c2f2f466bbf768545ca4687822206a63983ad

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=scrub/index-only/tpcc/w=1000 PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1231772&tab=buildLog

```
The test failed on master:
	cluster.go:1329,tpcc.go:169,cluster.go:1667,errgroup.go:57: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1231772-scrub-index-only-tpcc-w-1000:5 -- ./workload run tpcc --warehouses=1000 --histograms=logs/stats.json --wait=false --tolerate-errors --ramp=5m0s --duration=2h0m0s {pgurl:1-4} returned:
		stderr:
		
		stdout:
		n.go:428  error in newOrder: dial tcp 10.142.0.195:26257: connect: connection refused
		  10m11s    13633            0.0            0.0      0.0      0.0      0.0      0.0 delivery
		  10m11s    13633            0.0            0.0      0.0      0.0      0.0      0.0 newOrder
		  10m11s    13633            0.0            0.0      0.0      0.0      0.0      0.0 orderStatus
		  10m11s    13633            0.0            0.0      0.0      0.0      0.0      0.0 payment
		  10m11s    13633            0.0            0.0      0.0      0.0      0.0      0.0 stockLevel
		  10m12s    13633            0.0            0.0      0.0      0.0      0.0      0.0 delivery
		  10m12s    13633            0.0            0.0      0.0      0.0      0.0      0.0 newOrder
		  10m12s    13633            0.0            0.0      0.0      0.0      0.0      0.0 orderStatus
		  10m12s    13633            0.0            0.0      0.0      0.0      0.0      0.0 payment
		  10m12s    13633            0.0            0.0      0.0      0.0      0.0      0.0 stockLevel
		: signal: killed
	scrub.go:83,cluster.go:1667,errgroup.go:57: dial tcp 35.231.14.158:26257: connect: connection refused
	cluster.go:1688,tpcc.go:179,scrub.go:58,test.go:1237: unexpected node event: 2: dead
	cluster.go:953,context.go:90,cluster.go:942,asm_amd64.s:522,panic.go:397,test.go:785,test.go:771,cluster.go:1688,tpcc.go:179,scrub.go:58,test.go:1237: dead node detection: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod monitor teamcity-1231772-scrub-index-only-tpcc-w-1000 --oneshot --ignore-empty-nodes: exit status 1 5: skipped
		1: dead
		3: 3115
		4: 3140
		2: dead
		Error:  2: dead

```

SHA: https://github.com/cockroachdb/cockroach/commits/134478e4dde16919eb86efb81fb22d8cce8a9701

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=scrub/index-only/tpcc/w=1000 PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1234680&tab=buildLog

```
The test failed on release-19.1:
	cluster.go:1329,tpcc.go:169,cluster.go:1667,errgroup.go:57: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1234680-scrub-index-only-tpcc-w-1000:5 -- ./workload run tpcc --warehouses=1000 --histograms=logs/stats.json --wait=false --tolerate-errors --ramp=5m0s --duration=2h0m0s {pgurl:1-4} returned:
		stderr:
		
		stdout:
		run.go:428  error in payment: dial tcp 10.142.0.34:26257: connect: connection refused
		   8m42s    13914            0.0            0.0      0.0      0.0      0.0      0.0 delivery
		   8m42s    13914            0.0            0.0      0.0      0.0      0.0      0.0 newOrder
		   8m42s    13914            0.0            0.0      0.0      0.0      0.0      0.0 orderStatus
		   8m42s    13914            0.0            0.0      0.0      0.0      0.0      0.0 payment
		   8m42s    13914            0.0            0.0      0.0      0.0      0.0      0.0 stockLevel
		   8m43s    13914            0.0            0.0      0.0      0.0      0.0      0.0 delivery
		   8m43s    13914            0.0            0.0      0.0      0.0      0.0      0.0 newOrder
		   8m43s    13914            0.0            0.0      0.0      0.0      0.0      0.0 orderStatus
		   8m43s    13914            0.0            0.0      0.0      0.0      0.0      0.0 payment
		   8m43s    13914            0.0            0.0      0.0      0.0      0.0      0.0 stockLevel
		: signal: killed
	test.go:1225: test timed out (11h49m6.648238536s)
	scrub.go:83,cluster.go:1667,errgroup.go:57: pq: server is not accepting clients
	cluster.go:1688,tpcc.go:179,scrub.go:58,test.go:1237: unexpected node event: 3: dead
	cluster.go:1405,cluster.go:1424,cluster.go:1528,cluster.go:968,asm_amd64.s:522,panic.go:397,test.go:785,test.go:771,cluster.go:1688,tpcc.go:179,scrub.go:58,test.go:1237: context canceled

```

SHA: https://github.com/cockroachdb/cockroach/commits/bf399d2677783dc1eea7f5ede6d4561f95c0ea10

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=scrub/index-only/tpcc/w=1000 PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1234662&tab=buildLog

```
The test failed on master:
	cluster.go:1329,tpcc.go:169,cluster.go:1667,errgroup.go:57: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1234662-scrub-index-only-tpcc-w-1000:5 -- ./workload run tpcc --warehouses=1000 --histograms=logs/stats.json --wait=false --tolerate-errors --ramp=5m0s --duration=2h0m0s {pgurl:1-4} returned:
		stderr:
		
		stdout:
		n.go:428  error in newOrder: dial tcp 10.142.0.104:26257: connect: connection refused
		  24m15s     8978            0.0            0.0      0.0      0.0      0.0      0.0 delivery
		  24m15s     8978            0.0            0.0      0.0      0.0      0.0      0.0 newOrder
		  24m15s     8978            0.0            0.0      0.0      0.0      0.0      0.0 orderStatus
		  24m15s     8978            0.0            0.0      0.0      0.0      0.0      0.0 payment
		  24m15s     8978            0.0            0.0      0.0      0.0      0.0      0.0 stockLevel
		  24m16s     8978            0.0            0.0      0.0      0.0      0.0      0.0 delivery
		  24m16s     8978            0.0            0.0      0.0      0.0      0.0      0.0 newOrder
		  24m16s     8978            0.0            0.0      0.0      0.0      0.0      0.0 orderStatus
		  24m16s     8978            0.0            0.0      0.0      0.0      0.0      0.0 payment
		  24m16s     8978            0.0            0.0      0.0      0.0      0.0      0.0 stockLevel
		: signal: killed
	scrub.go:83,cluster.go:1667,errgroup.go:57: pq: server is not accepting clients
	cluster.go:1688,tpcc.go:179,scrub.go:58,test.go:1237: unexpected node event: 3: dead
	cluster.go:1405,cluster.go:1424,cluster.go:1528,cluster.go:968,asm_amd64.s:522,panic.go:397,test.go:785,test.go:771,cluster.go:1688,tpcc.go:179,scrub.go:58,test.go:1237: context canceled

```

SHA: https://github.com/cockroachdb/cockroach/commits/509c5b130fb1ad0042beb74e083817aa68e4fc92

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=scrub/index-only/tpcc/w=1000 PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1237002&tab=buildLog

```
The test failed on release-19.1:
	cluster.go:1329,tpcc.go:169,cluster.go:1667,errgroup.go:57: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1237002-scrub-index-only-tpcc-w-1000:5 -- ./workload run tpcc --warehouses=1000 --histograms=logs/stats.json --wait=false --tolerate-errors --ramp=5m0s --duration=2h0m0s {pgurl:1-4} returned:
		stderr:
		
		stdout:
		run.go:428  error in newOrder: dial tcp 10.128.0.9:26257: connect: connection refused
		   6m43s    14049            0.0            0.0      0.0      0.0      0.0      0.0 delivery
		   6m43s    14049            0.0            0.0      0.0      0.0      0.0      0.0 newOrder
		   6m43s    14049            0.0            0.0      0.0      0.0      0.0      0.0 orderStatus
		   6m43s    14049            0.0            0.0      0.0      0.0      0.0      0.0 payment
		   6m43s    14049            0.0            0.0      0.0      0.0      0.0      0.0 stockLevel
		   6m44s    14049            0.0            0.0      0.0      0.0      0.0      0.0 delivery
		   6m44s    14049            0.0            0.0      0.0      0.0      0.0      0.0 newOrder
		   6m44s    14049            0.0            0.0      0.0      0.0      0.0      0.0 orderStatus
		   6m44s    14049            0.0            0.0      0.0      0.0      0.0      0.0 payment
		   6m44s    14049            0.0            0.0      0.0      0.0      0.0      0.0 stockLevel
		: signal: killed
	cluster.go:1688,tpcc.go:179,scrub.go:58,test.go:1237: unexpected node event: 3: dead
	cluster.go:953,context.go:90,cluster.go:942,asm_amd64.s:522,panic.go:397,test.go:785,test.go:771,cluster.go:1688,tpcc.go:179,scrub.go:58,test.go:1237: dead node detection: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod monitor teamcity-1237002-scrub-index-only-tpcc-w-1000 --oneshot --ignore-empty-nodes: exit status 1 5: skipped
		3: dead
		4: 3090
		2: 3102
		1: 3338
		Error:  3: dead

```

SHA: https://github.com/cockroachdb/cockroach/commits/dd7c697e986fc528da7b12c6c10dcce7f64a486c

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=scrub/index-only/tpcc/w=1000 PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1252804&tab=buildLog

```
The test failed on release-19.1:
	cluster.go:1329,tpcc.go:168,cluster.go:1667,errgroup.go:57: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1252804-scrub-index-only-tpcc-w-1000:5 -- ./workload run tpcc --warehouses=1000 --histograms=logs/stats.json --wait=false --tolerate-errors --ramp=5m0s --duration=2h0m0s {pgurl:1-4} returned:
		stderr:
		
		stdout:
		un.go:428  error in payment: dial tcp 10.142.0.115:26257: connect: connection refused
		   5m19s    12971            0.0            0.0      0.0      0.0      0.0      0.0 delivery
		   5m19s    12971            0.0            0.0      0.0      0.0      0.0      0.0 newOrder
		   5m19s    12971            0.0            0.0      0.0      0.0      0.0      0.0 orderStatus
		   5m19s    12971            0.0            0.0      0.0      0.0      0.0      0.0 payment
		   5m19s    12971            0.0            0.0      0.0      0.0      0.0      0.0 stockLevel
		   5m20s    12971            0.0            0.0      0.0      0.0      0.0      0.0 delivery
		   5m20s    12971            0.0            0.0      0.0      0.0      0.0      0.0 newOrder
		   5m20s    12971            0.0            0.0      0.0      0.0      0.0      0.0 orderStatus
		   5m20s    12971            0.0            0.0      0.0      0.0      0.0      0.0 payment
		   5m20s    12971            0.0            0.0      0.0      0.0      0.0      0.0 stockLevel
		: signal: killed
	cluster.go:1688,tpcc.go:178,scrub.go:58,test.go:1237: unexpected node event: 3: dead
	cluster.go:953,context.go:89,cluster.go:942,asm_amd64.s:522,panic.go:397,test.go:785,test.go:771,cluster.go:1688,tpcc.go:178,scrub.go:58,test.go:1237: dead node detection: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod monitor teamcity-1252804-scrub-index-only-tpcc-w-1000 --oneshot --ignore-empty-nodes: exit status 1 5: skipped
		3: dead
		4: 4112
		2: 4005
		1: 4041
		Error:  3: dead

```

--- a/pkg/cmd/roachtest/scrub.go
+++ b/pkg/cmd/roachtest/scrub.go
@@ -25,13 +25,13 @@ import (
 )
 
 func registerScrubIndexOnlyTPCC(r *registry) {
-	// numScrubRuns is set to 5 assuming a single SCRUB run (index only) takes ~20 min
-	r.Add(makeScrubTPCCTest(5, 1000, time.Hour*2, "index-only", 5))
+	// numScrubRuns is set assuming a single SCRUB run (index only) takes ~1 min
+	r.Add(makeScrubTPCCTest(5, 100, 30*time.Minute, "index-only", 20))
 }
 
 func registerScrubAllChecksTPCC(r *registry) {
-	// numScrubRuns is set to 3 assuming a single SCRUB run (all checks) takes ~35 min
-	r.Add(makeScrubTPCCTest(5, 1000, time.Hour*2, "all-checks", 3))
+	// numScrubRuns is set assuming a single SCRUB run (all checks) takes ~2 min
+	r.Add(makeScrubTPCCTest(5, 100, 30*time.Minute, "all-checks", 10))
 }
 
 func makeScrubTPCCTest(

